src/main/java/com/backbase/communication/model/Attachment.java

```diff
@@ -0,0 +1,11 @@
+package com.backbase.communication.model;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Attachment {
```

Comment:
This is an intermediate model and it is suitable while we want to support multiple schemas, otherwise we can skip this model and just use generated one